Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -7718,18 +7718,13 @@ class BBQJIT {
         // We'll directly use these when passing parameters, since no other instructions we emit here should
         // overwrite registers currently occupied by values.
 
-        Vector<Value, N> resolvedArguments;
-        resolvedArguments.reserveInitialCapacity(arguments.size());
-        for (unsigned i = 0; i < arguments.size(); i ++) {
-            if (arguments[i].isConst())
-                resolvedArguments.uncheckedAppend(arguments[i]);
-            else
-                resolvedArguments.uncheckedAppend(Value::pinned(arguments[i].type(), locationOf(arguments[i])));
-
+        Vector<Value, N> resolvedArguments = WTF::map<N>(arguments, [&](auto& argument) {
```

Comment:
Is there a a reason we don't use `auto` in cases like this?

Reply:
I thought it made it more clear that the resulting vector has inline capacity but I don't mind going with `auto` at all.